Products/Zuul/routers/__init__.py

```diff
@@ -174,7 +174,14 @@ def asyncGetTree(self, id=None, additionalKeys=()):
             childData = Marshaller(child).marshal(keys)
```

Comment:
I was looking at `childData['children'] = []` and thinking it was `childData = []`
:+1: